components/card/src/vwc-card.ts

```diff
@@ -63,37 +63,42 @@ export class VWCCard extends LitElement {
 	}
 
 	protected override render(): unknown {
-		const actionsClassMap = {
-			'no-content': !(this.shouldShowActionsSlot)
+		const footerClassMap = {
+			'no-content': !(this.shouldShowFooterSlot)
 		};
 		return html`
 			<div class="vwc-card">
 				<div class="vwc-card-media">
 					<slot name="media"></slot>
```

Comment:
Was it our choice to name it as media? Is it the same as graphics?

Reply:
that part was media when I stated working on the card. Never touched it... Its basically for inserting a unique icon or image rather than one of vivid-icons

Reply:
we should probably align with list item's _graphic_

<img width="820" alt="image" src="https://user-images.githubusercontent.com/15140652/160396706-c14cef69-5f2d-4d12-8e59-0952f274d247.png">